ext/Errno/t/Errno.t

```diff
@@ -14,29 +14,33 @@ BAIL_OUT("No errno's are exported") unless @Errno::EXPORT_OK;
 my $err = $Errno::EXPORT_OK[0];
 my $num = &{"Errno::$err"};
 
-is($num, &{"Errno::$err"});
+is($num, &{"Errno::$err"},
+    'element in @Errno::EXPORT_OK found via sub call');
 
 $! = $num;
-ok(exists $!{$err});
+ok(exists $!{$err}, 'error found in %!');
 
 $! = 0;
-ok(! $!{$err});
+ok(! $!{$err}, 'no true value for error in %!');
 
-ok(join(",",sort keys(%!)) eq join(",",sort @Errno::EXPORT_OK));
+ok(join(",",sort keys(%!)) eq join(",",sort @Errno::EXPORT_OK),
+    'keys of %! match keys of @Errno::EXPORT_OK');
 
 eval { exists $!{[]} };
-ok(! $@);
+ok(! $@, "no error from most recent 'eval'");
```

Comment:
If you look back at the commit that introduced this test, it's checking that exists isn't throwing an exception when the key isn't a sub ref or a name.

Maybe `'$!{[]} doesn't throw an exception'` would be better.

Reply:
Is this the commit which you were referring to?
```
commit f8ed3476f18eeb645feeb028cc05ab8e72bae078
Author:     Rafael Garcia-Suarez <rgarciasuarez@gmail.com>
AuthorDate: Mon Oct 25 15:31:21 2004 +0000
Commit:     Rafael Garcia-Suarez <rgarciasuarez@gmail.com>
CommitDate: Mon Oct 25 15:31:21 2004 +0000

    Fix [perl #32130] Errno.pm must not pass references to "prototype"
```
If so, could I use that commit message as the basis for a better test description?

Reply:
Sure, but realize it isn't testing whether references are being passed to prototype (a user of the module doesn't care about that the implementation), but whether `$!{...}` is throwing an exception.

Reply:
Please review ac7e1be4ec.  If it's okay, I'll squash and merge.  Thanks.